### ISSUE TITLE
Out of Memory

### ISSUE THREAD

**Issue by lucaskingjade** [lisiyao21/Bailando#26]
Hi there,
When I run the second step as your instructions, I met the "out of memory" problem. 
I tried to debug the problem and found it is because the music_data is float64 and the memory is consumed rapidly when converting the list music_data to music_np. (in File "utils/functional.py").
Have you ever met the same problem like me? 
Is it possible to use float32 for training data(music_np) without decreasing the performance of the final model?

**Comment by lisiyao21** [comment]
Yes! It is feasible to do so. I am sorry for I did not clean the code sufficiently.
